lib/curv/lib/curves.curv

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
If you `include lib.curves`, then you import definitions of dot2, cross2 and sdQuadraticBezier, which pollutes your namespace.

A better design is to hide the internal code like this:
```
let
  dot2 = ...;
  cross2 = ...;
  sdQuadraticBezier = ...;
in {
  quadratic = ...;
}